src/renderer/src/lib/modal/ModalInstallForm.svelte

```diff
@@ -15,8 +15,7 @@
 	const client = trpc();
 	const modalStore = getModalStore();
 
-	const installedApps = client.getInstalledApps.createQuery();
-
+	const installedApps = client.getInstalledApps.createQuery(true);
```

Comment:
Interesting, why is the boolean needed as an arg now?

Reply:
The boolean says whether headless apps (i.e. apps without UI) should be fetched as well. In this case they should because it's now also officially supported to install headless apps through that dialog.